styles.css

```diff
@@ -494,3 +495,130 @@ button:disabled {
   width: 225.46px;
   height: 281.46px;
 }
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 58px;
+  width: 327px;
+  height: 604px;
+  margin: 0;
+}
+
+.primary-text {
+  width: 327px;
+  height: 216px;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin: 0%;
+}
+
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
  - Your project is not responsive on a mobile screen at the screen width of `375px`; due to that, you have a horizontal scroll. Make sure that your project is responsive and has the design the as the Figma design at a mobile screen width of `375px`
 
| Your Design |Figma Design |
| :---:   | :---: | 
| ![Screenshot from 2023-01-17 17-28-57](https://user-images.githubusercontent.com/92303891/212971183-fcf1d97c-7687-4eb7-8332-c0595b7a991d.png)| ![Screenshot from 2023-01-17 17-36-46](https://user-images.githubusercontent.com/92303891/212971591-471e2e5e-3221-4598-8f14-93cb4d275608.png)|